sql_metadata/token.py

```diff
@@ -142,21 +193,28 @@ def get_nth_previous(self, level: int):
             if level > 1:
                 return self.previous_token.get_nth_previous(level=level - 1)
             return self.previous_token
-        return None  # pragma: no cover
-
-
-EmptyToken = SQLToken(
-    value="",
-    is_keyword=False,
-    is_name=False,
-    is_punctuation=False,
-    is_dot=False,
-    is_wildcard=False,
-    is_integer=False,
-    is_float=False,
-    is_left_parenthesis=False,
-    is_right_parenthesis=False,
-    last_keyword=None,
-    subquery_level=0,
-    position=-1,
-)
+        return EmptyToken  # pragma: no cover
+
+    def find_token(
+        self,
+        value: Union[Union[str, bool], List[Union[str, bool]]],
+        direction: str = "left",
+        value_attribute: str = "value",
+    ) -> "SQLToken":
```

Comment:
I'm thinking about a bit better name here, maybe:

```suggestion
    def find_nearest_token(
        self,
        value: Union[Union[str, bool], List[Union[str, bool]]],
        direction: str = "left",
        value_attribute: str = "value",
    ) -> "SQLToken":
```